core-blocks/freeform/edit.js

```diff
@@ -84,6 +84,13 @@ export default class FreeformEdit extends Component {
 
 		this.editor = editor;
 
+		// Disable TinyMCE's keyboard shortcut help.
+		editor.on( 'BeforeExecCommand', ( event ) => {
+			if ( event.command === 'WP_Help' ) {
+				event.preventDefault();
+			}
+		} );
```

Comment:
Wondering if `editor.shortcuts.remove( 'ctrl+alt+h' )` might be a nicer way of doing this? Not sure.

https://www.tiny.cloud/docs/api/tinymce/tinymce.shortcuts/

Reply:
Doesn't seem to work unfortunately. I also thought `editor.editorCommands.addCommand( 'WP_Help', noop  )` might work but it doesn't. It suggests a race condition - that the command is added after freeforms setupEditor runs.

Reply:
Does what we're doing in RichText help?

https://github.com/WordPress/gutenberg/blob/4a78caed70cf899fa7bcd42258adbcafe5d5dda7/packages/editor/src/components/rich-text/index.js#L169

Reply:
Sadly not. Guessing it works there because Redo is already bound by TinyMCE at this point.